packages/flutter/lib/src/material/bottom_app_bar.dart

```diff
@@ -88,12 +99,15 @@ class BottomAppBar extends StatefulWidget {
   /// value is non-negative.
   ///
   /// If this property is null then [BottomAppBarTheme.elevation] of
-  /// [ThemeData.bottomAppBarTheme] is used. If that's null, the default value
-  /// is 8.
+  /// [ThemeData.bottomAppBarTheme] is used. If that's null and
+  /// [ThemeData.useMaterial3] is true, than the default value is 3 else is 8.
   final double? elevation;
 
   /// The notch that is made for the floating action button.
   ///
+  /// Material 3 doesn't support the notch, so this property has no effect on [BottomAppBar]
```

Comment:
I think the comment here should be removed.

Yes the standard M3 BottomAppBar is with a FAB.endContainer and no notch. But users can still set a docked FAB with notch so the notch should still be supported in M3.

Reply:
Removed